src/controllers/boardController.js

```diff
@@ -135,9 +127,6 @@ function updateGame(req, res) {
 function getAllBoards(_, res) {
 	//For this sprint, all users can see all games
 	return Board.getAllBoards().then((data) => {
-		for (let i = 0; i < data.length; i++) {
```

Comment:
Is the canonical link set elsewhere?

Reply:
Should be... `getAllBoards` calls the `Board` constructor which sets that